sechub-server/src/main/resources/application-postgres.properties

```diff
@@ -4,5 +4,9 @@
 ## --------------------------------------------
 spring.datasource.driverClassName=org.postgresql.Driver
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
-# next lines will be changed by kubernetes deployment by ENV entries - so its only used for local development
-spring.datasource.url=jdbc:postgresql://localhost:30432/sechub
+
+# next lines are defined for integration tests using PostgreSQL test container
+# for production you must change it by setting environment entries. E.g. in k8s deployment
+spring.datasource.url=${POSTGRES_DB_URL}
```

Comment:
Does this mean, people using SecHub have to update their deployment configuration?

Reply:
Before we had:
```
# next lines will be changed by kubernetes deployment by ENV entries - so its only used for local development
spring.datasource.url=jdbc:postgresql://localhost:30432/sechub
```
So this was alredy thought to be only used for local development.

With this issue, we added
```
spring.datasource.url=${POSTGRES_DB_URL}
spring.datasource.username=${POSTGRES_DB_USERNAME}
spring.datasource.password=${POSTGRES_DB_PASSWORD}
```
But maybe we should mention this in our documentation.

Reply:
mentioned inside documentation